docs/user_manual/source/rvfi.rst

```diff
@@ -223,11 +223,13 @@ For |corev|, the ``rvfi_mem`` interface has been expanded to support multiple me
    output [NRET * NMEM * XLEN/8 - 1 : 0] rvfi_mem_wmask
    output [NRET * NMEM * XLEN - 1 : 0]   rvfi_mem_rdata
    output [NRET * NMEM * XLEN - 1 : 0]   rvfi_mem_wdata
+   output [NRET * NMEM * 3    - 1 : 0]   rvfi_mem_prot
 
 Instructions will populate the ``rvfi_mem`` outputs with incrementing ``NMEM``, starting at ``NMEM=1``.
 
 Instructions with a single memory operation (e.g. all RV32I instructions), including split misaligned transfers, will only use NMEM = 1.
 Instructions with multiple memory operations (e.g. the push and pop instructions from Zcmp) use NMEM > 1 in case multiple memory operations actually occur.
+``rvfi_mem_prot`` indicates the value of OBI prot used for the memory access or accesses. Note that this will be undefined upon access faults.
```

Comment:
Next PRs keep documentation updates in a separate PR (that will result in a cleaner changelog)

Reply:
Sorry, I'll keep that in mind.